### PR TITLE
Davidk/tsk 16422 group auto generated automations into packs

### PR DESCRIPTION
**Pull request by DavidKorochik**
I didn't add tests because I don't think that it's worth testing the generation of the packs.
It's the same as we don't test the generation of the basic automations, only the helper functions.
About the pack description - this is the description that Oded chose.

### CODE REVIEW

File: components/library_automations/generator/packs/packs.go [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+			Description: fmt.Sprintf("This pack contains actions of %s.", action.CollectionDisplayName),
+			Metadata: models.Metadata{
+				ID:       uuid.NewV5(uuid.NamespaceURL, action.DisplayName+action.CollectionName).String(),
+				Level:    consts.LibFreeLevel,
+				SlugName: slugName,
+				Rank:     consts.HighSearchPriority,
+			},
+		}
+
+		filePath = filepath.Join(packsFolderPath, fmt.Sprintf("%s.yaml", slugName))
```
Comment (DanielDinsh) [inline components/library_automations/generator/packs/packs.go:33]: extract this, and the `slugName`, from the loop

File: components/library_automations/generator/packs/packs.go [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+	var filePath string
+
+	for _, action := range actions {
+		name := action.CollectionDisplayName + consts.DefaultBasicAutomationsPacksSuffix
+		slugName := text.GetSlugName(action.CollectionName + consts.DefaultBasicAutomationsPacksSuffix)
+		pack = models.Pack{
+			Name:        name,
+			Description: fmt.Sprintf("This pack contains actions of %s.", action.CollectionDisplayName),
+			Metadata: models.Metadata{
+				ID:       uuid.NewV5(uuid.NamespaceURL, action.DisplayName+action.CollectionName).String(),
```
Comment (DanielDinsh) [inline components/library_automations/generator/packs/packs.go:26]: why are you using `action.DisplayName` and not `FullName`?

File: components/library_automations/generator/packs/packs.go [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+				Level:    consts.LibFreeLevel,
+				SlugName: slugName,
+				Rank:     consts.HighSearchPriority,
```
Comment (DanielDinsh) [inline components/library_automations/generator/packs/packs.go:29]: HishPriority? (just checking I have no idea)

Comment (DavidKorochik) [reply components/library_automations/generator/packs/packs.go:29]: @DanielDinsh I checked in the packs in the library-content repo and the rank was 0

File: components/library_automations/generator/packs/packs.go [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+			Description: fmt.Sprintf("This pack contains actions of %s.", action.CollectionDisplayName),
+			Metadata: models.Metadata{
+				ID:       uuid.NewV5(uuid.NamespaceURL, action.FullName+action.CollectionName).String(),
+				Level:    consts.LibFreeLevel,
+				SlugName: slugName,
+				Rank:     consts.HighSearchPriority,
+			},
+		}
+
+	}
```
Comment (DanielDinsh) [inline components/library_automations/generator/packs/packs.go:33]: why do we need this loop?
you are creating 1 Pack

**Review by DanielDinsh — APPROVED** [review]

**Pull request merged**